libsolidity/CompilerStack.h

```diff
@@ -30,6 +30,10 @@
 #include <libdevcore/Common.h>
 #include <libdevcore/FixedHash.h>
 
+#if defined(__MINGW32__)
```

Comment:
Fix reason in commit message:

> Updates for mingw support. #undef conflicting defines on windows.

I can also reference in code if desired. In addition, i see you guys include windows in your builds so I'm not quite sure why it's failing on MinGW unless it's due to the include order of header files differing from standard mscv vs c++.